_includes/why.html

```diff
@@ -3,20 +3,14 @@
 
   <h2 id="why" class="section-header">Why</h2>
 
-  <p>
-    We would like to help improve diversity and the gender ratio in the Cape Town tech community.
-  </p>
+  <p>We want to push for all kinds of diversity in the Cape Town tech community: gender, race, sexual orientation, ability, and class. Women were the first population that RailsBridge focused on, but aren't the last.</p>
```

Comment:
Could we have ethnicity (an observed phenomenon) instead of race (a dangerous myth)?

I didn't like ability at first, but writing out criticism convinced me that it's good. It covers differently abled people and people with different education.

Class rattles my cage so much, I had trouble coming up with the thing you're pointing to. Is that economic class? I'd love a gentler word for that.